docs/source/geomodel/specifications/v0_1.md

```diff
@@ -0,0 +1,290 @@
+# GeoModel Version 0.1 Specification
```

Comment:
What if we renamed this to `docs/source/geomodel.md`? Then we could add a reference to it in the docs so that it would show up in https://mozdef.readthedocs.io/en/latest/, maybe under the `Usage` or `Advanced Settings`?